<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommitViewWindow</class>
 <widget class="QDialog" name="CommitViewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Commit View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ReadOnlyLineEdit" name="lineEdit_message"/>
      </item>
      <item>
       <widget class="ReadOnlyLineEdit" name="lineEdit_id"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QListWidget" name="listWidget_files"/>
     <widget class="FileDiffWidget" name="widget_diff" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReadOnlyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ReadOnlyLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>FileDiffWidget</class>
   <extends>QWidget</extends>
   <header location="global">FileDiffWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
